### DISCUSSION TITLE
unmapped reads too large

### DISCUSSION THREAD

**Discussion by ryrl9703** [alexdobin/STAR#1462] [Q&A]
Hello, everyone. When i use star to align the RNA seq date to human genome, the output of the unmapped.out.mate is always large, nd it unique mapped is about 70%. Is it normal? 

STAR \
			--genomeDir $index \
                        --genomeLoad NoSharedMemory \
                        --runThreadN $4 \
                        --sjdbOverhang 99 \
                        --sjdbGTFfile $gtf \
                        --alignIntronMin 20 \
                        --alignSJoverhangMin 8 \
                        --alignSJDBoverhangMin 1 \
                        --alignIntronMax 1000000 \
                        --alignMatesGapMax 1000000 \
                        --twopassMode Basic \
                        --quantMode GeneCounts \
                        --runMode alignReads \
                        --readFilesCommand zcat \
                        --readFilesIn $2/$id.fastq.gz \
                        --outSAMunmapped None \
                        --outSAMattributes All \
                        --outFilterType BySJout \
                        --outReadsUnmapped Fastx \
                        --outFilterMultimapNmax 1 \
                        --outFilterMismatchNmax 999 \
                        --outFileNamePrefix $3/$id \
                        --outSAMstrandField intronMotif \
                        --outFilterMismatchNoverLmax 0.04 \
                        --outSAMtype BAM SortedByCoordinate

**Comment by alexdobin** [comment]
Cross-post: https://github.com/alexdobin/STAR/issues/1463
